src/main/java/frc/trigon/robot/subsystem/pitcher/PitcherConstants.java

```diff
@@ -0,0 +1,40 @@
+package frc.trigon.robot.subsystem.pitcher;
+
+import com.ctre.phoenix.motorcontrol.TalonSRXFeedbackDevice;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+public class PitcherConstants {
+    static final int
+            MIN_TICKS = -3284,
+            MAX_TICKS = -1563;
+    static final double GEAR_RATIO = 10;
+    private static final int MOTOR_ID = 3;
+    final static WPI_TalonSRX MOTOR = new WPI_TalonSRX(MOTOR_ID);
+    private static final double
+            P = 3,
+            I = 0,
+            D = 0,
```

Comment:
we will test later but some d will probably be nice

Reply:
this is after testing. the p is very high but we compensate with peak_output

Reply:
retest with adding some d (very small like 0.0001) d always makes life better except when it doesn't

Reply:
@laviRZ told me to dont use d

Reply:
This works pretty good as it is. Instead of using D to limit the speed, we limit the maximum output from the PID.